README.md

```diff
@@ -5,18 +5,6 @@ This example demonstrates how to use the Mbed OS file system.
 
 You can find more information about the Mbed OS file system and other related pieces of the Mbed OS storage stack [in the storage overview](https://os.mbed.com/docs/latest/reference/storage.html).
 
-**Table of contents:**
```

Comment:
why are we removing the table?

Reply:
It's a fairly short document to navigate, and the table of content adds extra maintenance. We don't have any in other examples.